examples/hmm_demo_gavin_learning_2.ipynb

```diff
@@ -0,0 +1,1210 @@
+{
```

Comment:
do we want to rename this notebook since `gavin_learning` is included there, given that this is a public repo?